### PR TITLE
Resilio.ResilioSync version 2.7.2

### PR DESCRIPTION
**Pull request by lfont**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/15771)

### CODE REVIEW

**Comment by KaranKad** [comment]
Change `PackageIdentifier` to `Resilio.ResilioSync`

File: manifests/r/Resilio,Inc/ResilioSync/2.7.2.1375/Resilio,Inc.ResilioSync.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Created using wingetcreate
+
+PackageIdentifier: Resilio,Inc.ResilioSync
```
Comment (ItzLevvie) [inline manifests/r/Resilio,Inc/ResilioSync/2.7.2.1375/Resilio,Inc.ResilioSync.installer.yaml:3]:
```suggestion
PackageIdentifier: Resilio.ResilioSync
```

File: manifests/r/Resilio,Inc/ResilioSync/2.7.2.1375/Resilio,Inc.ResilioSync.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: Resilio,Inc.ResilioSync
```
Comment (ItzLevvie) [inline manifests/r/Resilio,Inc/ResilioSync/2.7.2.1375/Resilio,Inc.ResilioSync.locale.en-US.yaml:3]:
```suggestion
PackageIdentifier: Resilio.ResilioSync
```

File: manifests/r/Resilio,Inc/ResilioSync/2.7.2.1375/Resilio,Inc.ResilioSync.yaml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# Created using wingetcreate
+
+PackageIdentifier: Resilio,Inc.ResilioSync
```
Comment (ItzLevvie) [inline manifests/r/Resilio,Inc/ResilioSync/2.7.2.1375/Resilio,Inc.ResilioSync.yaml:3]:
```suggestion
PackageIdentifier: Resilio.ResilioSync
```

File: manifests/r/Resilio,Inc/ResilioSync/2.7.2.1375/Resilio,Inc.ResilioSync.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: Resilio,Inc.ResilioSync
+PackageVersion: 2.7.2.1375
```
Comment (ItzLevvie) [inline manifests/r/Resilio,Inc/ResilioSync/2.7.2.1375/Resilio,Inc.ResilioSync.locale.en-US.yaml:4]:
```suggestion
PackageVersion: 2.7.2
```

File: manifests/r/Resilio,Inc/ResilioSync/2.7.2.1375/Resilio,Inc.ResilioSync.yaml [hunk 5]
```diff
@@ -0,0 +1,7 @@
+# Created using wingetcreate
+
+PackageIdentifier: Resilio,Inc.ResilioSync
+PackageVersion: 2.7.2.1375
```
Comment (ItzLevvie) [inline manifests/r/Resilio,Inc/ResilioSync/2.7.2.1375/Resilio,Inc.ResilioSync.yaml:4]:
```suggestion
PackageVersion: 2.7.2
```

File: manifests/r/Resilio,Inc/ResilioSync/2.7.2.1375/Resilio,Inc.ResilioSync.installer.yaml [hunk 6]
```diff
@@ -0,0 +1,24 @@
+# Created using wingetcreate
+
+PackageIdentifier: Resilio,Inc.ResilioSync
+PackageVersion: 2.7.2.1375
```
Comment (ItzLevvie) [inline manifests/r/Resilio,Inc/ResilioSync/2.7.2.1375/Resilio,Inc.ResilioSync.installer.yaml:4]:
```suggestion
PackageVersion: 2.7.2
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**